.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug Report
+description: Something went awry and you'd like to tell us about it.
+labels: [t¹ 🐞 bug, triage]
```

Comment:
This should be `t⁸ ⋅ triage` here and in the other templates for consistency.

Reply:
ah yep. updated